docs/cookbook.rst

```diff
@@ -1,151 +0,0 @@
-Connexion Cookbook
-==================
-
-This section aims to be a cookbook of possible solutions for specific
-use cases of Connexion.
-
-Wildcard path parameters
-------------------------
-
-Path parameters cannot contain slashes by default, but sometimes it's useful
-to have a path parameter which takes the full remainder of the HTTP path
-including slashes, e.g. to allow parsing "my/deep/path" from
-"/pages/my/deep/path". Connexion supports parsing such path remainders
-by using ``format: path``:
-
-.. code-block:: yaml
-
-    paths:
-      /pages/{path}:
-        get:
-         # (...)
-          parameters:
-            - name: "path"
-              in: path
-              description: "Remainder of path, including slashes."
-              schema:
-                type: string
-                format: path
-
-
-Custom type format
```

Comment:
I don't think we have any documentation elsewhere on this, perhaps it could be interesting to keep a small reference in another part. E.g. in the `Validation` section?

Reply:
Good catch, I dismissed it too quickly, thinking this was about path parameter formats, which I already added to the routing documentation.